first_problem_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test/unit'
+require_relative'./first_problem'
+
+class FirstProblemTest < Test::Unit::TestCase
+
+  def test_compute_empty_array_0
+    array = Array.new
+    assert_equal(-1,FirstProblem.new(array).compute)
```

Comment:
Not very good at number theory but shouldn't this be `0`?

Reply:
i was trying to push for a "refactor" kind of step. Seems rather weird ideea now. Don't know exactly what i was thinking
But yes , it's 0...